tools/SourceKit/CMakeLists.txt

```diff
@@ -508,6 +512,24 @@ include_directories(BEFORE
 )
 
 if("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
+  if(SWIFT_BUILD_SOURCEKIT)
+    add_custom_command(
+      OUTPUT "${SWIFT_PATH_TO_LIBDISPATCH_BUILD}/src/.libs/libdispatch.so"
+      COMMAND /bin/sh ${SWIFT_PATH_TO_LIBDISPATCH_SOURCE}/autogen.sh
+      COMMAND /bin/sh ${SWIFT_PATH_TO_LIBDISPATCH_SOURCE}/configure
+      COMMAND /usr/bin/make
```

Comment:
Just want to make sure that there is no way the libdispatch.so you are building here is going to get mixed into the "real" build of libdispatch.so that probably happens later.  The configure command is missing the --with-swift-toolchain --prefix and --with-build-variant arguments that are supplied when it is build from swift/utils/build-script-impl